src/util.ts

```diff
@@ -524,6 +524,43 @@ export function expandMultilink(link: string): string {
 	return url.href
 }
 
+export function combineRaffleLink(links: string[]): string {
+	let firstCValue = null
```

Comment:
Would be great to have this validation inside createMultiLinkFromLinks instead of here, so that it happens for other multilinks too